lgc/patch/FragColorExport.cpp

```diff
@@ -142,8 +142,11 @@ Value *FragColorExport::handleColorExportInstructions(Value *output, unsigned hw
 
   switch (expFmt) {
   case EXP_FORMAT_32_R:
+    channelWriteMask = 0x1;
```

Comment:
This is broken -- channelWriteMask gets overwritten as it falls through to the code below. Clang gives a warning:
```
/Users/tpr/amd/git/sw/drivers/xgl/icd/api/compiler/lgc/patch/FragColorExport.cpp:128:3: warning: unannotated fall-through between switch labels [-Wimplicit-fallthrough]
  case EXP_FORMAT_32_GR:
  ^
/Users/tpr/amd/git/sw/drivers/xgl/icd/api/compiler/lgc/patch/FragColorExport.cpp:128:3: note: insert 'LLVM_FALLTHROUGH;' to silence this warning
  case EXP_FORMAT_32_GR:
  ^
  LLVM_FALLTHROUGH; 
/Users/tpr/amd/git/sw/drivers/xgl/icd/api/compiler/lgc/patch/FragColorExport.cpp:128:3: note: insert 'break;' to avoid fall-through
  case EXP_FORMAT_32_GR:
  ^
  break; 
```

Reply:
Thank you for pointing out. I will fix it.

Reply:
The fix is at https://github.com/GPUOpen-Drivers/llpc/pull/2800.